tests/confighelpertest.py

```diff
@@ -23,8 +23,7 @@
 import ConfigParser
 import logging
 import unittest
-import confighelper
-from confighelper import ValidationException
+from parkbenchcommon import confighelper
```

Comment:
Similar to what we do in Java, maybe we can "from parkbenchcommon.confighelper import ValidationException"? What do most Python programs do?